02.calendar/cal.rb

```diff
@@ -0,0 +1,24 @@
+require 'date'
+require 'optparse'
+
+params = ARGV.getopts("y:","m:")
+
+year = (params["y"] || Date.today.year).to_i # str型だとDate.new出来ないので
+month = (params["m"] || Date.today.month).to_i
+
+start_day = Date.new(year,month,1) # カレンダーで最初に表示する日
```

Comment:
```suggestion
start_day = Date.new(year, month, 1) # カレンダーで最初に表示する日
```

のようにカンマの後ろはスペースを入れるのが一般的です